static/src/javascripts/projects/common/modules/identity/api.js

```diff
@@ -0,0 +1,284 @@
+// @flow
+
+/* global escape:true */
+import { ajax } from 'lib/ajax';
+import config from 'lib/config';
+import { getCookie as getCookieByName } from 'lib/cookies';
+import mediator from 'lib/mediator';
+import { local } from 'lib/storage';
+import { mergeCalls } from 'common/modules/asyncCallMerger';
+
+/**
+ * Left this as an object as there are only static methods
+ * We'll need to change this once there is some state change
+ * TODO(jamesgorrie): Allow this to show policies too (not needed yet)
+ */
+
+let userFromCookieCache = null;
+
+const Id = {
+    cookieName: 'GU_U',
+    signOutCookieName: 'GU_SO',
+    fbCheckKey: 'gu.id.nextFbCheck',
+    lastRefreshKey: 'identity.lastRefresh',
+    idApiRoot: null,
+};
```

Comment:
I have checked where the properties on this object are being used. 

Only `lastRefreshKey` is used, in [`cookierefresh`](https://github.com/guardian/frontend/blob/master/static/src/javascripts-legacy/projects/common/modules/identity/cookierefresh.js#L12). Since it is not being used in this module, I think it can be safely inlined in `cookierefresh` and deleted from here.

The remaining properties can be removed from the `Id` object and replaced with `const`s? There's no need to group them like this. We also don't need to export them at the end.